CMakeLists.txt

```diff
@@ -93,8 +93,13 @@ elseif("${DDC_Kokkos_DEPENDENCY_POLICY}" STREQUAL "INSTALLED")
 	find_package(Kokkos CONFIG REQUIRED)
 endif()
 
+# Custom cmake modules
+list( APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake" )
+if("${CMAKE_VERSION}" VERSION_LESS 3.18)
+	list( APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/3.18" )
+endif()
```

Comment:
Now that we have upgraded docker images, we no longer need this 3.18 fix. Can you remove the associated files ?